tools/docker/README.md

```diff
@@ -18,38 +18,35 @@ _**All commands mentioned in this document should be run from the base Jetpack d
 
 ### Prerequisites
 
-* [Docker](https://hub.docker.com/search/?type=edition&offering=community)
-* [NodeJS](https://nodejs.org)
-* [Pnpm](https://pnpm.io/)
-* Optionally [Ngrok](https://ngrok.com) client and account or some other service for creating a local HTTP tunnel. It’s fine to stay on the free pricing tier with Ngrok.
+To use Jetpack's Docker environment, you will need:
 
-Install prerequisites; you will need to open up Docker to install its dependencies.
+- A [local copy of the Jetpack repository](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md##clone-the-repository).
+- [Docker](https://hub.docker.com/search/?type=edition&offering=community) installed and running.
+- [Jetpack's required tools](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#installing-development-tools).
 
-Start by cloning the Jetpack repository:
+Our Docker instance comes with a default settings file. You can modify those defaults by copying the file:
 
-```sh
-git clone git@github.com:Automattic/jetpack.git && cd jetpack
-```
-
-Optionally, copy settings file to modify it:
 ```sh
 cp tools/docker/default.env tools/docker/.env
```

Comment:
Note this only works right with docker-compose 1.28+. With earlier versions, the `.env` file needs to be in the monorepo root instead, although we might be able to work around that in the Jetpack CLI (see docs [here](https://docs.docker.com/compose/env-file/)).

Line 63 below refers to the monorepo-root location, BTW. That bit also seems like it may be confused into thinking there are two different `.env` files, which there aren't.

Reply:
Good call. Done in f6dfe635e01511f78e148920bf4e8e24c068227c